components/webui/client/src/pages/SearchPage/SearchControls/Presto/BuildSqlTestingInputs.tsx

```
@@ -67,6 +100,15 @@ const BuildSqlTestingInputs = () => {
             <input name={"sortItemList"}/>
             <label>limit:</label>
             <input name={"limitValue"}/>
+            <br/>
+            <label>database_name:</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)